src/Consolonia.Core/Infrastructure/AnsiConsoleOutput.cs

```diff
@@ -267,18 +268,21 @@ public void SetCaretStyle(CaretStyle caretStyle)
         public void HideCaret()
```

Comment:
We should think how to name it to understand what is immediate and what's buffered